contracts/auth/MultisigBase.sol

```diff
@@ -42,37 +42,7 @@ contract MultisigBase is IMultisigBase {
      * @dev Given the early void return, this modifier should be used with care on functions that return data.
      */
     modifier onlySigners() {
-        if (!signers.isSigner[msg.sender]) revert NotSigner();
-
-        bytes32 topic = keccak256(msg.data);
-        Voting storage voting = votingPerTopic[signerEpoch][topic];
-
-        // Check that signer has not voted, then record that they have voted.
-        if (voting.hasVoted[msg.sender]) revert AlreadyVoted();
-
-        voting.hasVoted[msg.sender] = true;
-
-        // Determine the new vote count.
-        uint256 voteCount = voting.voteCount + 1;
-
-        // Do not proceed with operation execution if insufficient votes.
-        if (voteCount < signers.threshold) {
-            // Save updated vote count.
-            voting.voteCount = voteCount;
-            return;
-        }
-
-        // Clear vote count and voted booleans.
-        voting.voteCount = 0;
-
-        uint256 count = signers.accounts.length;
-
-        for (uint256 i; i < count; ++i) {
-            voting.hasVoted[signers.accounts[i]] = false;
-        }
-
-        emit MultisigOperationExecuted(topic);
-
+        if (_onlySigners()) return;
```

Comment:
```suggestion
        if (!_onlySigners()) return;
```
switch the return value around so that the naming makes more sense?

Reply:
Yeah makes sense, will change the name to _isFinalSignerVote too as per @re1ro's suggestion